src/couch_replicator/src/couch_replicator_doc_processor.erl

```diff
@@ -843,7 +1019,7 @@ started_worker(_Id) ->
     1 == meck:num_calls(couch_replicator_doc_processor_worker, spawn_worker, 4).
 
 removed_job(Id) ->
-    meck:called(couch_replicator_scheduler, remove_job, [test_rep(Id)]).
+    meck:called(couch_replicator_scheduler, remove_job, [Id]).
```

Comment:
Note: here before we were passing the wrong time to remove_job function. The function should take a replication ID `{"123", "+continuous"}` not a `#rep{}` record.